testcases/OpTestXive.py

```diff
@@ -0,0 +1,209 @@
+           if(temp != 'y'):
+               self.fail("xive is not configured properly")
+        res = self.cv_HMC.vterm_run_command(self.console, "cat /etc/os-release | grep NAME | head -1")
+        if 'SLES' in res[0].strip():
+            self.distro = 'SLES'
+            self.cv_HMC.vterm_run_command(self.console, "cat /etc/default/grub|grep xive")
+            self.cv_HMC.vterm_run_command(self.console, "sed -i '/xive=/c\GRUB_CMDLINE_LINUX_DEFAULT=\"xive=off\"' /etc/default/grub")
+            self.cv_HMC.vterm_run_command(self.console, "grub2-mkconfig -o /boot/grub2/grub.cfg")
+            self.cv_HMC.vterm_run_command(self.console, "cat /etc/default/grub | grep xive")
+            self.cv_HMC.vterm_run_command(self.console, "cat /boot/grub2/grub.cfg | grep xive")
+            self.cv_HMC.vterm_run_command(self.console, "sync;sleep 10")
+            self.cv_HMC.restart_lpar()
+            self.cv_HMC.wait_login_prompt(self.console, username="root", password="passw0rd")
+            self.console = self.cv_HMC.get_console()
+            self.cv_HMC.vterm_run_command(self.console, "which stty && stty cols 300; which stty && stty rows 30;")
+            self.cv_HMC.vterm_run_command(self.console, "cat /proc/cmdline")
+            self.cv_HMC.vterm_run_command(self.console,"cat /proc/interrupts | grep -Eai 'xive'")
+            self.cv_HMC.vterm_run_command(self.console, "sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT=\"xive=off\"/GRUB_CMDLINE_LINUX_DEFAULT=\"xive=on\"/' /etc/default/grub")
+            self.cv_HMC.vterm_run_command(self.console, "grub2-mkconfig -o /boot/grub2/grub.cfg")
+            self.cv_HMC.vterm_run_command(self.console, "cat /etc/default/grub | grep xive")
+            self.cv_HMC.vterm_run_command(self.console, "sync;sleep 10")
+            self.cv_HMC.restart_lpar()
+            self.cv_HMC.wait_login_prompt(self.console, username="root", password="passw0rd")
+            self.console = self.cv_HMC.get_console()
+            self.cv_HMC.vterm_run_command(self.console, "which stty && stty cols 300; which stty && stty rows 30;")
+            self.cv_HMC.vterm_run_command(self.console, "cat /proc/cmdline")
+            self.cv_HMC.vterm_run_command(self.console,"cat /proc/interrupts | grep -Eai 'xive'")
+        elif 'Red Hat' in res[0].strip():
+            self.distro = 'RHEL'
+            self.cv_HMC.vterm_run_command(self.console, "which stty && stty cols 300; which stty && stty rows 30;")
+            self.cv_HMC.vterm_run_command(self.console, "grubby --info=/boot/vmlinuz-`uname -r`")
+            self.cv_HMC.vterm_run_command(self.console, "grubby --remove-args=xive* --update-kernel=/boot/vmlinuz-`uname -r`")
+            self.cv_HMC.vterm_run_command(self.console, "grubby --info=/boot/vmlinuz-`uname -r`")
+            self.cv_HMC.vterm_run_command(self.console, "grubby --args=xive=off --update-kernel=/boot/vmlinuz-`uname -r`")
```

Comment:
can we make use of `update_kernel_cmdline()` in OpTestInstallUtil.py ? may be the API needs some changes to support powervm.